_data/en.yml

```diff
@@ -177,49 +178,25 @@ faq:
   hTradeCost: "How much does it cost to trade on Bisq?"
   pTradeCost: >
     <p>To trade on Bisq, users pay (1) trading fees to Bisq and (2) mining fees to miners.</p>
 
-    <strong>Trading fees</strong> <p>Trading fees are payable in BTC or BSQ. BSQ is colored bitcoin that Bisq uses to fund and govern itself. It's optional to use, but there are <a href="https://docs.bisq.network/getting-started-dao-traders.html" target="_blank">significant benefits</a>.</p> <div id="faq-tables"> <p class="note">BTC trading fees, per 1 BTC traded:</p> <table> <thead> <tr> <th></th> <th>BTC fees</th> <th>Fee as % of 1 BTC trade size</th> </tr> </thead> <tbody> <tr> <td>Maker fee</td> <td>0.002 BTC</td> <td>0.20%</td> </tr> <tr> <td>Taker fee</td> <td>0.006 BTC</td> <td>0.60%</td> </tr> </tbody> </table> <p class="note">BSQ trading fees, per 1 BTC traded:</p> <table> <thead> <tr> <th></th> <th>BSQ fees</th> <th>Fee as % of 1 BTC trade size</th> </tr> </thead> <tbody> <tr> <td>Maker fee</td> <td>13 BSQ</td> <td>0.13%</td> </tr> <tr> <td>Taker fee</td> <td>38 BSQ</td> <td>0.38%</td> </tr> </tbody> </table> </div> <p>BSQ fee percentages above assume a 0.00001 BSQ/BTC rate.</p> <p>Nominal BTC and BSQ fees are adjusted periodically by DAO voting to accommodate for market fluctuations.</p> <p>The minimum trading fees are set at 0.00005 BTC and 0.03 BSQ to avoid dust limits.</p>
+    <strong>Trading fees</strong> <p>Trading fees are payable in BTC or BSQ. BSQ is colored bitcoin that Bisq uses to fund and govern itself. It's optional to use, but there are <a href="https://docs.bisq.network/getting-started-dao-traders.html" target="_blank">significant benefits</a>.</p> <div id="faq-tables"> <p class="note">BTC trading fees, per 1 BTC traded:</p> <table> <thead> <tr> <th></th> <th>BTC fees</th> <th>Fee as % of 1 BTC trade size</th> </tr> </thead> <tbody> <tr> <td>Maker fee</td> <td>0.002 BTC</td> <td>0.20%</td> </tr> <tr> <td>Taker fee</td> <td>0.006 BTC</td> <td>0.60%</td> </tr> </tbody> </table> <p class="note">BSQ trading fees, per 1 BTC traded:</p> <table> <thead> <tr> <th></th> <th>BSQ fees</th> <th>Fee as % of 1 BTC trade size</th> </tr> </thead> <tbody> <tr> <td>Maker fee</td> <td>10 BSQ</td> <td>0.10%</td> </tr> <tr> <td>Taker fee</td> <td>30 BSQ</td> <td>0.30%</td> </tr> </tbody> </table> </div> <p>BSQ fee percentages above assume a 0.00001 BSQ/BTC rate.</p> <p>Nominal BTC and BSQ fees are adjusted periodically by DAO voting to accommodate for market fluctuations.</p> <p>The minimum trading fees are set at 0.00005 BTC and 0.03 BSQ to avoid dust limits.</p>
```

Comment:
Shouldn't it be 0.0001 BSQ/BTC rate instead of 0.00001

Reply:
Ah, yes -- updated.